### PR TITLE
Add requests spec for Job

### PR DESCRIPTION
**Pull request by chechetov**
Here I added requests spec for Job controller.

### CODE REVIEW

File: spec/requests/jobs_spec.rb [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+    end
+
+    before do
+        sign_in user
+
+    end
+
+    describe "GET /index" do
+        it "renders a successful response" do
+            get jobs_url(jub_id: job.id)
```
Comment (itsAndL) [inline spec/requests/jobs_spec.rb:33]: you just need "jobs_url" not "jobs_url(jub_id: job.id)"

File: spec/requests/jobs_spec.rb [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+            expect(response).to be_successful
+        end
+    end
+
```
Comment (itsAndL) [inline spec/requests/jobs_spec.rb:37]: you're gonna need to add test for show, new, and edit actions

File: spec/requests/jobs_spec.rb [hunk 3]
```diff
@@ -0,0 +1,95 @@
...
+        context "With valid params" do
+            it "creates a new job" do
+            expect {
+                post jobs_url, params: { job: valid_attributes }
+            }.to change(Job, :count).by(1)
+            end
+            it "redirects to jobs index page" do
+                post jobs_url, params: { job: valid_attributes }
+                expect(response).to redirect_to(jobs_url)
+                expect(flash[:notice]).to eq(I18n.t("flash.successfully_created"))
```
Comment (itsAndL) [inline spec/requests/jobs_spec.rb:48]: make the flash independent on the redirect (make a new unit test or scenario)

File: spec/requests/jobs_spec.rb [hunk 4]
```diff
@@ -0,0 +1,95 @@
...
+                patch "/jobs/#{job.id}", params: {job: valid_attributes}
+                job.reload
+
+                expect(job.title).to eq("Test Job")
+                expect(job.location).to eq("Test Location")
+                expect(job.overview).to eq("Test Overview")
+                expect(job.min_salary).to eq(1)
+                expect(job.max_salary).to eq(2)
+                expect(job.unit).to eq("hour")
+            end
```
Comment (itsAndL) [inline spec/requests/jobs_spec.rb:83]: add tests for redirection, and flash

File: spec/requests/jobs_spec.rb [hunk 5]
```diff
@@ -0,0 +1,95 @@
...
+        end
+        context "with invalid parameters" do
+            it "does not update the requested Job" do
+                patch "/jobs/#{job.id}", params: {job: invalid_attributes}
+                expect(response).to have_http_status(:unprocessable_entity)
+            end
+        end
+    end
+
+    # Describe delete?
```
Comment (itsAndL) [inline spec/requests/jobs_spec.rb:93]: Add test to the destroy action

File: spec/requests/jobs_spec.rb (hunk 1 above)
Comment (chechetov) [reply spec/requests/jobs_spec.rb:33]: done

**Pull request merged**